File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/TreeViews_ViewModelMyParentIndexRowHandlesRow.cs
/// <filename>
///     TreeViews_ViewModelMyParentIndexRowHandlesRow.cs
/// </filename>
namespace widgetassertions
{
    public class TreeViews_ViewModelMyParentIndexRowHandlesRow
    {
        private string ValuesLabelText;

        public virtual string getValuesLabelText()
        {
            return this.ValuesLabelText;
        }

        private int RowIndex;

        public virtual int getRowIndex()
        {
            return this.RowIndex;
        }

        private int ParentRowIndex;

        public virtual int getParentRowIndex()
        {
            return this.ParentRowIndex;
        }
    }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/TreeViews_ViewModelMyParentIndexRowHandlesRow.cpp
#include "TreeViews_ViewModelMyParentIndexRowHandlesRow.h"
#include <string>

namespace widgetassertions
{
  std::string TreeViews_ViewModelMyParentIndexRowHandlesRow::getValuesLabelText()
  {
    return this->ValuesLabelText;
  }
  int TreeViews_ViewModelMyParentIndexRowHandlesRow::getRowIndex()
  {
    return this->RowIndex;
  }
  int TreeViews_ViewModelMyParentIndexRowHandlesRow::getParentRowIndex()
  {
    return this->ParentRowIndex;
  }
}
